README.md

```diff
@@ -31,6 +31,15 @@ find research data from the RKI.
 For further details, please consult our
 [project page](https://www.rki.de/DE/Content/Forsch/MEx/MEx_node.html).
 
+**Contact** \
+For more information, please feel free to email us at [mex@rki.de](mailto:mex@rki.de).
+
+### Publisher of this document
+**Robert Koch-Institut** \
+Nordufer 20 \
+13353 Berlin \
+Germany
+
```

Comment:
```suggestion
```

we can remove this change from here, because it will be applied automatically by cruft and would only cause conflicts if it were here